diff --git a/node_modules/react-native-reanimated/android/CMakeLists.txt b/node_modules/react-native-reanimated/android/CMakeLists.txt
index 1111111..2222222 100644
--- a/node_modules/react-native-reanimated/android/CMakeLists.txt
+++ b/node_modules/react-native-reanimated/android/CMakeLists.txt
@@ -108,6 +108,7 @@ endif()
 
 target_link_libraries(
   reanimated
+  c++_shared
   log
   ReactAndroid::reactnative
   ReactAndroid::jsi
