diff --git a/node_modules/react-native-worklets/android/CMakeLists.txt b/node_modules/react-native-worklets/android/CMakeLists.txt
index 1111111..2222222 100644
--- a/node_modules/react-native-worklets/android/CMakeLists.txt
+++ b/node_modules/react-native-worklets/android/CMakeLists.txt
@@ -88,7 +88,7 @@ include_directories(
 # build shared lib
 set_target_properties(worklets PROPERTIES LINKER_LANGUAGE CXX)
 
-target_link_libraries(worklets android log ReactAndroid::reactnative
+target_link_libraries(worklets c++_shared android log ReactAndroid::reactnative
                       ReactAndroid::jsi fbjni::fbjni)
 
 if(ReactAndroid_VERSION_MINOR GREATER_EQUAL 82)
